hub/management/commands/import_new_constituencies.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaType
+
+
+class Command(BaseCommand):
+    help = "Import basic area information for new constituencies"
+
+    data_file = settings.BASE_DIR / "data" / "new_constituencies.csv"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-q", "--quiet", action="store_true", help="Silence progress bars."
+        )
+
+    def handle(self, quiet: bool = False, *args, **options):
+        if not quiet:
+            print("Importing Areas")
+        df = pd.read_csv(self.data_file)
+        area_type, created = AreaType.objects.get_or_create(
+            name="2023 Parliamentary Constituency",
+            code="WMC23",
```

Comment:
It looks like we’ve made a decision, elsewhere (eg: https://pages.mysociety.org/2025-constituencies/) to call these "2025" constituencies. So should we use `2025 Parliamentary Constituency` and `WMC25` here? /cc @ajparsons